### DISCUSSION TITLE
iphone上打开任何一个视频都报TypeError，电脑win10 chrome正常

### DISCUSSION THREAD

**Discussion by dovela** [AlistGo/alist#1714] [Q&A]
System error: TypeError: undefined is not a constructor (evaluating 'new window.MediaSource')

![F009D344-6A7C-41D5-BC51-745007879EE8](https://user-images.githubusercontent.com/33590269/190924102-522d595b-e1d1-43dd-b6da-736968acf675.jpeg)

alist v3版，能给个排查错误的方向建议吗

**Comment by fuyunyiqie** [comment]
解决了吗？我也是
